src/Azure.Functions.Cli/Helpers/PythonHelpers.cs

```diff
@@ -97,9 +97,9 @@ public static void AssertPythonVersion(WorkerLanguageVersionInfo pythonVersion,
             // Python 3.x (but not 3.6 or 3.7), not recommended, may fail
             if (pythonVersion.Major == 3)
             {
-                if (errorIfNotSupported) throw new CliException($"Python 3.6.x or 3.7.x is required for this operation. "
-                + "Please install Python 3.6 or 3.7, and use a virtual environment to switch to Python 3.6 or 3.7.");
-                ColoredConsole.WriteLine(WarningColor("Python 3.6.x or 3.7.x is recommended, and used in Azure Functions."));
+                if (errorIfNotSupported) throw new CliException($"Python 3.6.x or 3.7.x is required for the version of Azure Functions you are using (2.x). "
+                + "Please see if another version of Functions supports your version (https://aka.ms/functions-python-versions). Otherwise, please install Python 3.6 or 3.7 and use a virtual environment to switch to Python 3.6 or 3.7.");
```

Comment:
Probably not a problem but the trailing parenthesis in the link breaks the link in the VS Code terminal. Looks like a fix is on the way (currently behind experimental flag). https://github.com/microsoft/vscode/issues/21125